<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Tufail-Vivo" failures="0" tests="3" name="Packagetesting.Class1" time="0.005" errors="1" timestamp="2020-06-25T18:49:15 EDT" skipped="0">
  <testcase classname="Packagetesting.Class1" name="demo" time="0.004"/>
  <system-out/>
  <testcase classname="Packagetesting.Class1" name="dim" time="0.000">
    <error message="
Parameter &#039;URL&#039; is required by @Test on method dim but has not been marked @Optional or defined
in C:\Users\ahmad\AppData\Local\Temp\testng-eclipse--1497629649\testng-customsuite.xml" type="org.testng.TestNGException">
      <![CDATA[org.testng.TestNGException: 
Parameter 'URL' is required by @Test on method dim but has not been marked @Optional or defined
in C:\Users\ahmad\AppData\Local\Temp\testng-eclipse--1497629649\testng-customsuite.xml
at org.testng.internal.Parameters.createParams(Parameters.java:268)
at org.testng.internal.Parameters.createParametersForMethod(Parameters.java:356)
at org.testng.internal.Parameters.createParameters(Parameters.java:700)
at org.testng.internal.Parameters.handleParameters(Parameters.java:875)
at org.testng.internal.Parameters.handleParameters(Parameters.java:740)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:788)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- dim -->
  <system-out/>
  <testcase classname="Packagetesting.Class1" name="secondClass" time="0.001"/>
  <system-out/>
</testsuite> <!-- Packagetesting.Class1 -->
